packages/utils/package.json

```diff
@@ -68,18 +68,15 @@
   },
   "dependencies": {
     "@eslint-community/eslint-utils": "^4.4.0",
-    "@types/json-schema": "^7.0.15",
-    "@types/semver": "^7.5.8",
```

Comment:
This line was initially removed in https://github.com/typescript-eslint/typescript-eslint/commit/2ce1c1d22#diff-21f8eb1d7a149d42ab0454ea905d44f624ff1d0aeb6d7a88d75b42962ca6b053